spec/lib/cli_spec.rb

```diff
@@ -10,13 +10,13 @@
   end
   context 'without config' do
     let(:argv) { ['-r', repo] }
-    it 'should raise an error' do
-      expect{ subject }.to raise_error(Sanctify::ScannerError)
+    it 'should not raise an error' do
+      expect{ subject }.not_to raise_error
```

Comment:
When removing the AWS secret access key matcher, this test started to fail because an error was no longer being raised. Investigating, it turns out that this was raising an error not because the configuration was not being provided, but because it was failing validation on the AWS secret access key it found in the repo.